### PR TITLE
[16.0][MIG] mgmtsystem_survey: Migration to 16.0

### PR DESCRIPTION
**Pull request by FernandoRomera**

### CODE REVIEW

File: mgmtsystem_survey/views/survey_survey.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (max3903) [inline mgmtsystem_survey/views/survey_survey.xml:1]:
```suggestion
```

**Review by max3903 — CHANGES_REQUESTED** [review]

**Review by ivantodorovich — APPROVED** [review]

**Review by AntoniRomera — APPROVED** [review]

File: mgmtsystem_survey/views/survey_survey.xml (hunk 1 above)
Comment (FernandoRomera) [reply mgmtsystem_survey/views/survey_survey.xml:1]: Same than #471

**Comment by FernandoRomera** [comment]
@max3903 
Can you resolve requested changes?

**Review by max3903 — APPROVED** [review]

**Pull request merged**